cypress/e2e/dfi.spec.ts

```diff
@@ -0,0 +1,158 @@
+        "section-title-building-better-tomorrow",
+        "BUILDING A BETTER TOMORROW WITH DFI"
+      );
+      cy.checkElementVisibilityAndText(
+        "harness-dfi-power-title",
+        "Harness the power of DFI"
+      );
+      cy.checkElementVisibilityAndText(
+        "harness-dfi-power-desc",
+        "A gateway to decentralized applications and tools, offering potential for greater returns, lower fees, and democratic governance."
+      );
+
+      cy.get("article > span.font-bold.mb-4")
+        .should("be.visible")
+        .and("have.length", 9);
+      if (viewport === "iphone-xr") {
+        cy.get("div[data-index='0']")
+          .should("have.attr", "aria-hidden")
+          .and("include", "false");
+        cy.get("div[data-index='1']")
+          .should("have.attr", "aria-hidden")
+          .and("include", "true");
+        cy.get("ul.harness-dfi-dots").scrollIntoView();
+        cy.get("ul.harness-dfi-dots > li > button")
+          .last()
+          .click({ scrollBehavior: false });
+        cy.get("div[data-index='0']")
+          .should("have.attr", "aria-hidden")
+          .and("include", "true");
+        cy.get("div[data-index='1']")
+          .should("have.attr", "aria-hidden")
+          .and("include", "false");
+      }
+    });
+
+    it("should have Purchase DFI easily section with links ", () => {
+      cy.checkElementVisibilityAndText(
+        "section-title-purchase-dfi",
+        "PURCHASE $DFI EASILY"
+      );
+      cy.checkElementVisibilityAndText(
+        "get-dfi-title",
+        "Get DFI on trusted exchanges"
+      );
+      cy.checkElementVisibilityAndText(
+        "get-dfi-desc",
+        "Use our partner links to securely buy DFI on trusted cryptocurrency exchanges and easily transfer them to your DeFiChain wallet address."
+      );
+    });
+
+    it("should have Trusted exchanges visible and clickable ", () => {
+      exchanges.forEach((exchangeItem) => {
+        cy.checkElementVisibilityAndHref(
+          `getdfisection-exchange-${exchangeItem.id}`,
+          exchangeItem.url
+        );
+      });
+    });
+
+    it("should have Read from Whitepaper section visible and clickable", () => {
+      cy.checkElementVisibilityAndText(
+        "section-title-transparent-distribution",
+        "TRANSPARENT DISTRIBUTION"
+      );
+      cy.checkElementVisibilityAndText(
+        "initial-token-dist-title",
+        "Initial Token Distribution"
+      );
+
+      cy.checkElementVisibilityAndHref(
+        "explore-dmc-button-read-from-whitepaper",
+        "/white-paper"
+      );
+    });
+
+    it("should have Initial supply and Masternodes sections visible ", () => {
+      cy.checkElementVisibilityAndText(
+        "initial-token-supply",
+        "INITIAL SUPPLY"
+      );
+      cy.checkElementVisibilityAndText(
+        "initial-token-masternodes-title",
+        "TO MASTERNODES OVER TIME"
+      );
+      cy.findByTestId("initial-token-dist-desc").should("be.visible");
```

Comment:
curious that we only assert the actual content of `initial-token-masternodes-desc` but not `initial-token-dist-desc`